src/mail.sh

```diff
@@ -130,6 +136,44 @@ function mail_send()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function gathers the email addresses that are present in the group
+# files, it first looks for the global files and then the local files.
+#
+# @group_name: Name of the group of emails
+#
+# Return:
+# list of emails separated by commas
+function get_group_addresses()
+{
+  local group_name="$1"
+  local addresses=''
+
+  addresses+="$(load_addresses "${KW_ETC_DIR}/mail_groups/${group_name}")"
+  addresses+="$(load_addresses "${PWD}/.kw/mail_groups/${group_name}")"
```

Comment:
Don't we need to check:
1. Files exist
2. Files have content
3. Content is well-formatted